sdk/loadtestservice/load-testing-rest/README.md

```diff
@@ -0,0 +1,192 @@
+
+#### Test
+
+A test specifies the test script, and configuration settings for running a load test. You can create one or more tests in an Azure Load Testing resource.
+
+#### App Component
+
+When you run a load test for an Azure-hosted application, you can monitor resource metrics for the different Azure application components (server-side metrics). While the load test runs, and after completion of the test, you can monitor and analyze the resource metrics in the Azure Load Testing dashboard.
+
+#### Metrics
+
+During a load test, Azure Load Testing collects metrics about the test execution. There are two types of metrics:
+
+1. Client-side metrics give you details reported by the test engine. These metrics include the number of virtual users, the request response time, the number of failed requests, or the number of requests per second.
+
+2. Server-side metrics are available for Azure-hosted applications and provide information about your Azure application components. Metrics can be for the number of database reads, the type of HTTP responses, or container resource consumption.
+
+### Test Engine
+
+A test engine is computing infrastructure that runs the Apache JMeter test script. You can scale out your load test by configuring the number of test engines. The test script runs in parallel across the specified number of test engines.
+
+### Test Run
+
+A test run represents one execution of a load test. It collects the logs associated with running the Apache JMeter script, the load test YAML configuration, the list of app components to monitor, and the results of the test.
+
+### Data-Plane Endpoint
+
+Data-plane of Azure Load Testing resources is addressable using the following URL format:
+
+`00000000-0000-0000-0000-000000000000.aaa.cnt-prod.loadtesting.azure.com`
+
+The first GUID `00000000-0000-0000-0000-000000000000` is the unique identifier used for accessing the Azure Load Testing resource. This is followed by  `aaa` which is the Azure region of the resource.
+
+The data-plane endpoint is obtained from Control Plane APIs.
+
+**Example:** `1234abcd-12ab-12ab-12ab-123456abcdef.eus.cnt-prod.loadtesting.azure.com`
+
+In the above example, `eus` represents the Azure region `East US`.
+
+## Examples
+
+### Creating a load test
+
+```javascript
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+
+var TEST_ID = "some-test-id";
+var DISPLAY_NAME = "my-load-test";
+
+const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
+
+await Client.path("/loadtests/{testId}", TEST_ID).patch({
+  contentType: "application/merge-patch+json",
+  body: {
+    displayName: DISPLAY_NAME,
+    description: "",
+    loadTestConfig: {
+      engineInstances: 1,
+      splitAllCSVs: false,
+    },
```

Comment:
Can we add sample values for secrets, environment variables and pass fail criteria as well?

Reply:
added.